core-bundle/src/Resources/public/core.js

```diff
@@ -1329,6 +1329,30 @@ var Backend =
 		});
 	},
 
+	/**
+	 * Retrieve the interactive help
+	 */
+	retrieveInteractiveHelp: function (elements) {
+		elements && elements.each(function (element) {
+			var title = element.retrieve('tip:title');
+			title && element.set('title', title);
+		});
+	},
+
+	/**
+	 * Hide the interactive help
+	 */
+	hideInteractiveHelp: function () {
+		var hideTips = function () {
+			document.querySelectorAll('.tip-wrap').forEach(function (tip) {
+				tip.setStyle('display', 'none');
+			});
+		};
+		hideTips();
+		// hide delayed tips - timeout: Tips.Contao.options.showDelay
+		setTimeout(hideTips, 1000);
```

Comment:
What if someone overwrites `Tips.Contao.options.showDelay` in their installation? Can we not use the variable instead of a fixed value here?

Reply:
I'm no javascript freak 😇 - maybe someone from @contao/developers can support.
I did not get the value of the comment!

Reply:
@ausi /cc

Reply:
`(new Tips.Contao()).options.showDelay`

The correct way would be to delete the actual timers of the tooltips with `clearTimeout()`, but I didn’t find a way to get access to them.

Reply:
Yes, the only way would be if we save the initiated tooltips while `addInteractiveHelp`. Currently we don't have access to them because of the closures e.g. https://github.com/contao/contao/blob/a70561370fcf46a6148c1ca899d1d48c5ec2d023/core-bundle/src/Resources/public/core.js#L1324

Reply:
By the way the delay variable is now set with the recommended code and I don't want to touch the initiation of tooltips to get a better solution for the delayed tooltips!